package.json

```diff
@@ -23,6 +23,7 @@
     "next-auth": "^4.24.8",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.3.0",
```

Comment:
dont add  additional package for icon, we already have lucide react and radix icon

Reply:
you can check the changes now @TenzDelek